DE.yaml

```
@@ -672,25 +675,25 @@ SPECIALIST_SHOUTMESSAGE_RESURRECTED: Die Seele deiner Spezialistenkarte wurde wi
 SPECIALIST_SHOUTMESSAGE_USE_NEEDED: Du musst in einen Spezialisten verwandelt sein, um dies zu tun!
 SUM_MESSAGE_FAILED: Summierung fehlgeschlagen.
 SUM_MESSAGE_SUCCESS: Summierung erfolgreich!
-TIMESPACE_CHATMESSAGE_EASY_MODE: The time-space is in easy mode - the duration has been extended and you will not receive selected rewards.
+TIMESPACE_CHATMESSAGE_EASY_MODE: Der Timespace ist im Einfachen-Modus - die verfügbare Zeit verlängert sich und du wirst deine ausgewählte Belohnung nicht erhalten.
 TIMESPACE_CHATMESSAGE_GATE_LOCKED: Die Tür ist verschlossen!
-TIMESPACE_CHATMESSAGE_LOW_LEVEL: My level is too low to join this time-space.
-TIMESPACE_CHATMESSAGE_NOT_FINISHED_TASKS: Not all time-space missions have been completed.
-TIMESPACE_CHATMESSAGE_ONLY_TEAM_LEADER_START: Nur der Gruppenleiter kann den Timespace beginnen
-TIMESPACE_CHATMESSAGE_TIME_SPACE_PENALTY: '{0}% of points reduced because of level penalty.'
-TIMESPACE_DIALOG_ASK_JOIN_TO_PLAYER: Do you want to join a group member's time-space?
-TIMESPACE_DIALOG_ASK_REVIVAL: Do you want to resurrect yourself?
+TIMESPACE_CHATMESSAGE_LOW_LEVEL: Mein Level ist zu niedrig, um diesen Timespace beizutreten.

```

```suggestion
TIMESPACE_CHATMESSAGE_LOW_LEVEL: Dein Level ist zu niedrig um den Timespace zu betreten.

```

---

```suggestion
TIMESPACE_CHATMESSAGE_LOW_LEVEL: Dein Level ist zu niedrig, um diesen Timespace zu betreten.

```

---

```suggestion
TIMESPACE_CHATMESSAGE_LOW_LEVEL: Mein Level ist zu niedrig, um diesen Timespace zu betreten.
```

english `My` => Speach bubble